SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorCloudLauncherSettings.h

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Engine/EngineTypes.h"
+#include "Misc/Paths.h"
+#include "SpatialGDKEditorSettings.h"
+#include "UObject/Package.h"
+
+#include "SpatialGDKEditorCloudLauncherSettings.generated.h"
+
+
+/**
+* Enumerates available Region Codes
+*/
+UENUM()
+namespace ERegionCode
+{
+	enum Type
+	{
+		US = 1,
+		EU,
+		AP,
+	};
+}
+
+UCLASS(config = SpatialGDKEditorCloudLauncherSettings, defaultconfig)
```

Comment:
I don't think there's a need for this to go into it's own file, can't we keep it in the `SpatialGDKEditorSettings.ini` ?

Reply:
That's definitely something we can change - will refactor and merge both files 👍.

Reply:
I've merged the files together and the settings reside in the `SpatialGDKEditorSettings.h` file now.